packages/backend-common/src/errors.ts

```diff
@@ -75,3 +75,8 @@ export class NotFoundError extends CustomErrorBase {}
  * resource.
  */
 export class ConflictError extends CustomErrorBase {}
+
+/**
+ * The requested resource has not changed since last request.
+ */
+export class NotModifiedError extends CustomErrorBase {}
```

Comment:
these have corresponding tests that check the mapping to HTTP error codes etc

Reply:
Sweet. :)